crate/kmip/src/kmip/kmip_operations.rs

```diff
@@ -1080,6 +1085,19 @@ pub struct ReKeyKeyPairResponse {
     pub public_key_unique_identifier: UniqueIdentifier,
 }
 
+#[derive(Debug, Serialize, Deserialize, Default)]
+#[serde(rename_all = "PascalCase")]
+pub enum ReKeyKeyPairAction {
+    #[default]
+    RotateAttributes,
```

Comment:
Why is it a goof idea to make this action the default? Do you really need a default in the first place?